webpack/assets/javascripts/react_app/components/Layout/LayoutSelectors.js

```diff
@@ -29,7 +29,7 @@ const patternflyItems = (data, currentLocation, currentOrganization) => {
     const childrenArray = [];
     item.children.forEach(child => {
       const childObject = {
-        id: `menu_item_${snakeCase(child.name)}`,
+        id: child.name ? `menu_item_${snakeCase(child.name)}` : null,
```

Comment:
/cc @glekner.

we are assuming child is always defined? how about ```child.name === undefined``` ? 

also, would you mind adding a test for it?

Reply:
`child.name` should always exist. we agreed that there will be no more blank dividers when the layout got merged.
its part of patternfly's restrictions now. I think we should change the translate helper  `translate as __` to be more defensive though

Reply:
@glekner I think the issue is coming from plugins who don't follow this yet, I think its best that core is defensive and does render the menu even if a plugin breaks it.

so if a child.name is not defined, we should just skip the entry ?

Reply:
skipping sounds like the right solution

Reply:
updated to skip items with no name, also changed code to use `.map` instead of pushing to arrays and extracted child to item logic to separate function.

Reply:
now with tests :)